app/views/ChargeSummary.scala.html

```diff
@@ -80,14 +108,33 @@
     </div>
 }
 
+@getBalancingPaymentsExplanationContent = {
+    <div>
+        @p(id = Some("lpi-bcd1")){@messages("chargeSummary.lpi.balancingCharge.p1")}
+        @p(id = Some("lpi-bcd2")) {
+            @messages("chargeSummary.lpi.balancingCharge.textOne")
+            <a class="govuk-link" href="https://www.bankofengland.co.uk/monetary-policy/the-interest-rate-bank-rate">@messages("chargeSummary.lpi.balancingCharge.linkText")</a>
+            @messages("chargeSummary.lpi.balancingCharge.textTwo")
+        }
+        @p(id = Some("lpi-bcd3")) {
+            @messages("chargeSummary.lpi.balancingCharge.p3")
+            <a class="govuk-link" href="https://www.gov.uk/government/publications/rates-and-allowances-hmrc-interest-rates-for-late-and-early-payments/rates-and-allowances-hmrc-interest-rates#current-late-payment-and-repayment-interest-rates">@messages("chargeSummary.lpi.balancingCharge.p3LinkText")</a><br><br>
+        }
+    </div>
+}
+
 @getExplanationContent = @{viewModel.documentType match {
-        case Poa1Charge | Poa2Charge    => paymentOnAccountExplanation
-        case BalancingCharge            => balancingChargeExplanation
+        case Poa1Charge | Poa2Charge if !viewModel.latePaymentInterestCharge => paymentOnAccountExplanation
+        case Poa1Charge | Poa2Charge if viewModel.latePaymentInterestCharge => getLpiPoaExplanationContent
+        case TRMNewCharge if !viewModel.latePaymentInterestCharge => balancingChargeExplanation //TRMNewCharge used instead of Balancing charge
```

Comment:
I would assume these comments is something we need to discuss / address?

Reply:
Yes, we should raise another ticket to possibly tidy up the charge types to be consistent over each type. But it doesn't pose a problem at the time being